File: python/Array/4344.py
import sys

if __name__ == "__main__":
    tc=int(input())
    for i in range(tc):
        input_list=list(map(int,input().split()))
        stu_num=input_list[0] #첫번째 수는 학생수 
        _list=input_list[1:] #인덱스 1번(2번째) 부터 끝까지 따로 slice
        avg=sum(_list)/stu_num
        count=0
        for val in _list:
            if(val>avg):
                count+=1
        result=count/stu_num *100
        print("{:.3f}%".format(result)) # 소수 3째자리 까지 출력

File: Go/Array/4344.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	var tc int
	reader := bufio.NewReader(os.Stdin)
	fmt.Scan(&tc)

	for i := 0; i < tc; i++ {

		str, _ := reader.ReadString('\n')
		list := strings.Fields(str)
		stu_num, _ := strconv.Atoi(list[0])
		_list := list[1:]
		avg := 0
		for _, val := range _list {
			temp, _ := strconv.Atoi(val)
			avg += temp
		}
		avg /= stu_num
		count := 0
		for _, val := range _list {
			temp, _ := strconv.Atoi(val)
			if temp > avg {
				count++
			}
		}
		result := float32(count) / float32(stu_num) * 100
		fmt.Printf("%.3f%s\n", result, "%")

	}

}
